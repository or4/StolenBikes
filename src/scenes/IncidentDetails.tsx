import React from 'react';
import * as R from 'ramda';

// @ts-ignore
export function IncidentDetails(props): React.ReactElement {
    // @ts-ignore
    return <div>IncidentDetails {R.path(['match', 'params', 'id'], props)}</div>;
}
