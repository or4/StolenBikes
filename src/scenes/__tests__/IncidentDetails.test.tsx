import React from 'react';
import ReactDOM from 'react-dom';
import { IncidentDetails } from '../IncidentDetails';

it('renders without crashing', (): void => {
    const div = document.createElement('div');
    ReactDOM.render(<IncidentDetails />, div);
    ReactDOM.unmountComponentAtNode(div);
});
