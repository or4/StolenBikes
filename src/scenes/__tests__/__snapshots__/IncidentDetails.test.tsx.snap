// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IncidentDetailsScene should render correctly 1`] = `
<Component
  dispatch={[Function]}
  incidents={Array []}
  match={
    Object {
      "params": Object {
        "id": 4,
      },
    }
  }
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "clear": [Function],
        "get": [Function],
        "notify": [Function],
        "subscribe": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": [Function],
    }
  }
/>
`;
